diff --git a/opal/util/minmax.h b/opal/util/minmax.h
index 5b7f3c768b..6a9e41822c 100644
--- a/opal/util/minmax.h
+++ b/opal/util/minmax.h
@@ -42,7 +42,7 @@ OPAL_DEFINE_MINMAX(float, float)
 OPAL_DEFINE_MINMAX(double, double)
 OPAL_DEFINE_MINMAX(void *, ptr)
 
-#if OPAL_C_HAVE__GENERIC
+#ifdef OPAL_C_HAVE__GENERIC
 #    define opal_min(a, b) \
         (_Generic((a) + (b),                                                \
                                  int8_t: opal_min_8,                    \
